openApiDocs/v1.0/Applications.yml

```diff
@@ -242,6 +242,13 @@ paths:
           schema:
             type: string
           x-ms-docs-key-type: application
+        - name: appId
+          in: path
+          description: 'Alternate key: appId of application'
+          style: simple
+          schema:
+            type: string
+            nullable: true
```

Comment:
The PR is introducing an Invalid appId parameter as an alternate key of application. This will lead to AutoREST generating an invalid `-AppId` parameter on `*-MgApplication`.

See related issue at https://github.com/microsoft/OpenAPI.NET.OData/issues/297.